### PR TITLE
Rj add authorization

### PR DESCRIPTION
**Pull request by jrzricardo**

### CODE REVIEW

**Review by jrzricardo — COMMENTED** [review]
looks ship shape!

File: app/views/comments/_comment.html.erb [hunk 1]
```diff
@@ -10,12 +10,14 @@
       <p><%= comment.body %></p>
     </div>
     <div>
-      <%= link_to edit_comment_path(comment), class: "btn btn-link btn-sm text-muted" do %>
-        <i class="fas fa-edit fa-fw"></i>
-      <% end %>
+      <% if current_user == comment.author %>
```
Comment (ghost) [inline app/views/comments/_comment.html.erb:13]: Ah! I'm not sure how I missed this in my code, but you did good not letting it slip by! Great work!

**Review by ghost — APPROVED** [review]
Clean, legible, and as mentioned you caught something I missed. Great work!

**Review by CarlosCastanedaDev — APPROVED** [review]
Great job, Ricardo!

**Pull request merged**